### PR TITLE
Feature/fix envs

### PR DESCRIPTION
**Pull request by mg-4xxi**

### CODE REVIEW

File: .env.dist [hunk 1]
```diff
@@ -1,27 +1,20 @@
 # App configuration
-ROBOT_EMAIL_ADDRESS=noemail@example.com
-ROBOT_EMAIL_NAME=Robot
+SENDER_ADDRESS=noemail@dev.caesar.team
```
Comment (beshkenadze) [inline .env.dist]: Please do not use real data in dist files.

File: .env.dist [hunk 2]
```diff
@@ -47,14 +40,13 @@ INVITATION_SALT=salt
 # Delivery is disabled by default via "null://localhost"
 MAILER_URL=null://localhost
 DELIVERY_ADDRESS=your_dev@email.com
-DELIVERY_NAME=Caesar
+COMPANY_NAME='Local Caesar'
```
Comment (beshkenadze) [inline .env.dist]: Use “Caesar App” instead of.

Comment (beshkenadze) [reply .env.dist]: Also I suggest use “APP_NAME” instead of “COMPANY_NAME”

Comment (mg-4xxi) [reply .env.dist]: done

File: .env.dist (hunk 1 above)
Comment (mg-4xxi) [reply .env.dist]: replaced

File: .env.dist (hunk 2 above)
Comment (mg-4xxi) [reply .env.dist]: > APP_NAME

done

**Pull request merged**